vendor-extra/ContentPageBundle/tests/Controller/ContentControllerTest.php

```diff
@@ -38,12 +40,12 @@ public function it_returns_the_title(Request $request, array $twigContext) : voi
                 [],
                 <<<XML
 <?xml version="1.0" encoding="UTF-8"?>
-<item xmlns="http://libero.pub">
-    <front xml:lang="en">
-        <id>id</id>
-        <title>Title</title>
-    </front>
-</item>
+<libero:item xmlns:libero="http://libero.pub">
```

Comment:
Have to use prefixes as `DOMNode::getNodePath()` doesn't support default namespaces and the element path ends up being `/*/*`. (Refs https://github.com/ThomasWeinert/FluentDOM/issues/61.)

Reply:
Looks like a big bug for anything real :thinking: 
e.g. this uses the default namespace so it won't work as-is?
```
$ curl http://unstable--dummy-api.libero.pub/scholarly-articles/items/article1/versions/latest
<?xml version="1.0" encoding="UTF-8"?>
<item xmlns="http://libero.pub">
    <front xml:lang="en">
        <id>article1</id>
        <title>Article 1</title>
    </front>
</item>
```

Reply:
Yep. Unlikely to use in outside tests, but will need to address the issue mentioned if we do.